backend/core/scripts/import-listings-basic.ts

```diff
@@ -7,9 +7,10 @@ import dbOptions = require("../ormconfig")
 import { Program } from "../src/program/entities/program.entity"
 import { AddressCreateDto } from "../src/shared/dto/address.dto"
 
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
`// eslint-disable-next-line @typescript-eslint/no-var-requires`
I'd prefer estline-disable-next-line here rather than disabling it for the whole file, so that we're notified if we do this again in this file